lib/activerecord-bitemporal/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module ActiveRecord
   module Bitemporal
-    VERSION = "2.0.0"
+    VERSION = "2.0.1"
```

Comment:
hi.
I think this update will be a minor version update, not a patch version, as this update will add features, not bug fixes. 
`2.1.0` ?